pkg/ccl/backupccl/restore_online.go

```diff
@@ -427,10 +432,6 @@ func checkManifestsForOnlineCompat(ctx context.Context, manifests []backuppb.Bac
 	if len(manifests) < 1 {
 		return errors.AssertionFailedf("expected at least 1 backup manifest")
 	}
-	// TODO(online-restore): Remove once we support layer ordering.
-	if len(manifests) > 1 {
```

Comment:
Maybe we should keep this check but change the `1` to maxOnlineLayers and set maxOnlineLayers to 3 or something tiny for now? I'm just worried that we have a lot of work and testing to do before we want to claim generalized support.